src/UuidGenerator.php

```diff
@@ -3,22 +3,22 @@
 namespace JDR\Uuid\Doctrine\ODM;
 
 use Doctrine\ODM\MongoDB\DocumentManager;
-use Doctrine\ODM\MongoDB\Id\AbstractIdGenerator;
+use Doctrine\ODM\MongoDB\Id\IdGenerator;
```

Comment:
`AbstractIdGenerator` was deprecated in `doctrine/mongodb-odm:2.1` and will be removed in `3.0`.